src/hub.html

```diff
@@ -400,9 +407,9 @@
                 <a-entity in-world-hud="haptic:#player-right-controller;raycaster:#player-right-controller;" rotation="30 0 0">
                     <a-rounded height="0.08" width="0.2" color="#000000" position="-0.30 0.125 0" radius="0.040" opacity="0.35" class="hud bg"></a-rounded>
                     <a-image scale="0.07 0.06 0.06" position="-0.23 0.165 0.001" src="#presence-count" material="alphaTest:0.1;"></a-image>
-                    <a-entity id="hud-presence-count" text="value:; width:1.1; align:center;" position="-0.155 0.165 0"></a-entity>
+                    <a-entity id="hud-presence-count" text="value:; width:1.1; align:center;" position="-0.155 0.165 0" matrix-auto-update="target: text;"></a-entity>
```

Comment:
the text component updates the transform of the underlying mesh based upon the contents -- so if the contents change, the matrix needs to be updated.

this is a good example of where this flag is preventing an optimization -- since this text is "slowly changing" I would argue regardless of the other concerns that it would be better for the text component to set the flag when the text changes.